manifests/r/RoyalApps/RoyalTS/6.1.50425/RoyalApps.RoyalTS.installer.yaml

```diff
@@ -0,0 +1,15 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
+
+PackageIdentifier: RoyalApps.RoyalTS
+PackageVersion: 6.1.50425

```

Comment:
```suggestion
PackageVersion: 6.1.50425.0
```

Reply:
I think this is a problem with wingetcreate command line. I specified the version 6.1.50425 (without the .0) but the installer is built (for other reasons) with the .0 at the end. I can commit this change for sure but I'm afraid the next time it runs in my pipeline, we have the same issue. Should I open an issue with wingetcreate for that?

Reply:
Your installer includes .0 in the DisplayVersion registry string, so if you specified without the .0 to wingetcreate then.. you are the issue?

Afaik wingetcreate uses your specified values unless its a new package.

Reply:
Correct, my bad. Will update my script.